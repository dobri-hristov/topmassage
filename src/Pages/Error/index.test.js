import React from 'react'
import TestRenderer from 'react-test-renderer'
import ErrorPage from './index.js'

describe('Error page', () => {
    test('should render correctly', () => {
        const component = TestRenderer.create(<ErrorPage />)
        const tree = component.toJSON()
        expect(tree).toMatchSnapshot()
    })
})