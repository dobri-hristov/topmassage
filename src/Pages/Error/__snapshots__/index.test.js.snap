// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error page should render correctly 1`] = `
<div
  className="container"
>
  <p
    className="icon"
  >
    <i
      class="far fa-frown"
    />
  </p>
  <h2
    className="line"
  >
    Страницата не е намерена
  </h2>
  <p
    className="line"
  >
    Изглежда не можем да намерим това, което търсите. Може би една от тези връзки ще ви помогне.
  </p>
  <div
    className="line"
  >
    <button
      className="container"
      onClick={[Function]}
    >
      Начало
    </button>
    <button
      className="container"
      onClick={[Function]}
    >
      Масажи
    </button>
    <button
      className="container"
      onClick={[Function]}
    >
      Цени
    </button>
    <button
      className="container"
      onClick={[Function]}
    >
      Контакти
    </button>
    <button
      className="container"
      onClick={[Function]}
    >
      Запази час
    </button>
    <button
      className="container"
      onClick={[Function]}
    >
      За нас
    </button>
    <button
      className="container"
      onClick={[Function]}
    >
      Вход
    </button>
    <button
      className="container"
      onClick={[Function]}
    >
      Регистрация
    </button>
  </div>
</div>
`;
